examples/kubernetes/modify-volume/manifests/pod-with-volume.yaml

```diff
@@ -32,7 +32,7 @@ spec:
   storageClassName: ebs-sc
   resources:
     requests:
-      storage: 100Gi
+      storage: 10Gi
```

Comment:
if you're going to change this, you should re-run the example and update the readme with correct outputs so it isn't mismatched